docker-image/Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM continuumio/miniconda3
+
+ENV WORKSPACE /workspace
+
+# Setup workspace environment
+RUN apt-get update && apt-get install -y gcc
+RUN conda install jupyter notebook
+RUN pip install --no-cache-dir syft numpy
```

Comment:
why do we need numpy seperately

Reply:
I got the installation instructions from Udacity and I thought that this is how it should be installed, didn't noticed the installation instructions here in the repo.

Reply:
just tested it without numpy and it's working allright. Making the change.